### PR TITLE
Auto-DJ Add to Control Objects 

### PR DESCRIPTION
**Pull request by daschuer**
Fixes https://bugs.launchpad.net/mixxx/+bug/1209153

### CODE REVIEW

File: src/controllers/controlpickermenu.cpp [hunk 1]
```diff
@@ -358,6 +358,14 @@ ControlPickerMenu::ControlPickerMenu(QWidget* pParent)
                        tr("Load Track Into Stopped Deck"),
                        tr("Load selected track into first stopped deck"),
                        m_libraryStr, libraryMenu);
+    addPrefixedControl("[Playlist]", "AutoDjAddTop",
+                       tr("Add to Auto DJ Queue (top)"),
+                       tr("Prepend selected track to the  Auto DJ Queue"),
```
Comment (esbrandt) [inline src/controllers/controlpickermenu.cpp]: Nit: stray whitespace in between ``the  Auto``

File: src/controllers/controlpickermenu.cpp [hunk 2]
```diff
@@ -358,6 +358,14 @@ ControlPickerMenu::ControlPickerMenu(QWidget* pParent)
...
+                       m_libraryStr, libraryMenu);
+    addPrefixedControl("[Playlist]", "AutoDjAddBottom",
+                       tr("Add to Auto DJ Queue (bottom)"),
+                       tr("Append the selected track to the  Auto DJ Queue"),
```
Comment (esbrandt) [inline src/controllers/controlpickermenu.cpp]: Nit: stray whitespace in between ``the  Auto``

File: src/controllers/controlpickermenu.cpp [hunk 3]
```diff
@@ -358,6 +358,14 @@ ControlPickerMenu::ControlPickerMenu(QWidget* pParent)
                        tr("Load Track Into Stopped Deck"),
                        tr("Load selected track into first stopped deck"),
                        m_libraryStr, libraryMenu);
+    addPrefixedControl("[Playlist]", "AutoDjAddTop",
```
Comment (esbrandt) [inline src/controllers/controlpickermenu.cpp]: In the library right-click menu we follow the ``add bottom`` and ``add top`` ordering.
In midi wizard it is ``Add top`` and ``Add bottom``

**Comment by esbrandt** [comment]
Thanks, works as expected. Tested with assigning midi and keyboard mappings.

**Comment by daschuer** [comment]
Thank you for review!

**Pull request merged**